### DISCUSSION TITLE
Can't turn off password protection for old application

### DISCUSSION THREAD

**Discussion by eghernqvist** [vercel/community#2140] [Help]
### Summary

I turned on password protection back when it was free and under the Security part of the settings; this has since been moved and I do not remember the credentials and want to turn off the protection now. I'm finding no way to do this though...
Anyone have any idea? I tried redeploying, but have yet to try a fresh deploy with a new commit.

### Additional information

![image](https://user-images.githubusercontent.com/10044687/233349507-ca22ce40-1c39-489e-85ca-01e75a7f922d.png)

![image](https://user-images.githubusercontent.com/10044687/233349630-c6082085-e7ab-4019-9507-170901105e4d.png)

![image](https://user-images.githubusercontent.com/10044687/233349566-75c494d9-1245-4225-891b-6ee52103cd82.png)

**Comment by swarnava** [comment]
Can you sure your project details such as domain or link of the project ? Typically redeploying to a project should disable the password protection
